pkg/operator/operator/validations.go

```diff
@@ -744,3 +786,53 @@ func getValidationK8sResources() ([]kunstructured.Unstructured, *kresource.Quant
 
 	return virtualServices, maxMem, err
 }
+
+func validateOverridenImages(images, keys []string) error {
+	images = slices.RemoveEmpties(images)
```

Comment:
This may not be desired now that the values will be set to the defaults. However, I do think we can skip the accessibility validation if the image is in the cortexlabs dockerhub domain (we should still check the version in the tag though)

Reply:
Yeah, this is already removed.